kcp/README.MD

```diff
@@ -1,22 +1,17 @@
 # Pipeline Services on managed KCP
 
 ## Clusters access
-The `login.sh -e staging` script will log you in into the various systems:
-1. KCP
-2. KCP workspace
-3. pipeline cluster
-4. ArgoCD
+Create a directory with the following files:
+* `argocd.yaml`: kubeconfig with the active context connecting to the argocd instance;
+* `kcp.yaml`: kubeconfig with the active context connecting to your KCP workspace;
+* `plnsvc.yaml`: the kubeconfig with with the active context connecting to the plnsvc cluster.
 
-The script will create various config files in `kcp/work`.
+This directory will referenced as `KUBECONFIG_DIR`.
 
-Some tokens have a limited lifetime. If you haven't login for a while, you might start
-seeing errors related to authentication/permissions. Running the script again will
-refresh the required credentials.
-
-See `login.sh --help` for available options.
-
-### ArgoCD
-No setup is required.
+The first time one of the deploy/test/reset command is run, use `-k $KUBECONFIG_DIR`.
+This will create a local copy of those files in `kcp/work/kubeconfig`. Those files will
+then be used by default for every command, until you overwrite their value by specifying
+a new `-k $KUBECONFIG_DIR` parameter.
```

Comment:
If the above suggestion is taken, then we can remove this.

Reply:
See my response to your comment.